starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -2937,6 +2943,26 @@ sub proseco_args {
         return \%proseco_args;
     }
 
+    my $man_angle_data = call_python("state_checks.get_obs_man_angle",
+        [ $targ_cmd->{tstop}, $self->{backstop} ]);
+    $targ_cmd->{man_angle_calc} = $man_angle_data->{'angle'};
+
+    if (defined $man_angle_data->{'warn'}){
+        push @{$self->{warn}}, $man_angle_data->{'warn'};
+    }
+
+    # Set a maneuver angle to be used by the proseco acquisition probability calculation.
+    # Here the goal is to be conservative and use the angle that is derived from the
+    # time in NMM before acquisition (the man_angle_calc from state_checks.get_obs_man_angle)
+    # if needed but not introduce spurious warnings for cases where the angle as derived
+    # from the time in NMM is in a neighboring bin for proseco maneuver error probabilities.
+    # To satisfy those goals, use the derived-from-NMM-time angle if it is more than 5 degrees
+    # different from the actual maneuver angle.  Otherwise use the actual maneuver angle.
+    # This also lines up with what is printed in the starcheck maneuver output.
+    my $man_angle = (abs($targ_cmd->{man_angle_calc} - $targ_cmd->{angle}) > 5)
```

Comment:
@taldcroft Does this work for you? It is a substantive update since your approval. I'm rerunning functional review with it in place.  To be fully conservative I suppose the other option would be to use the angle-from-nmm-time only if 5 or more degrees *greater* than the maneuver angle (instead of just different).

Reply:
If the angle-from-nmm-time is more than 5 degrees SMALLER than the maneuver angle, that would indicate a bug in the code somewhere. In that case there should be a critical warning and the review should use the maneuver angle.

Reply:
Nice comment BTW!!